### PR TITLE
DCOS 2.2 GA Docs

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-71751

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 1]
```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
...
+- Marathon migration failed while upgrading from 1.13.7 to 2.0.2. (COPS-5897)
+- Nested Marathon groups were not inheriting enforceRole behavior from top-level groups. (COPS-6529)
+- enforceRole was automatically being set to false on all Marathon groups. (COPS-6533)
+- Selecting *Run* on a job or selecting *Delete* to remove a group had no effect and threw an exception. (COPS-6324) 
+- etcd provided an incorrect response in calicoctl. (COPS-6341)  
+- Users were unable to remove empty folders from Metronome. (COPS-6139) 
+- Hostname Resolution was failing for VIPs. (COPS-6411)  
+- Using file-based secrets caused mount failure and issues in the json editor. (COPS-6085, D2IQ-68114)  
+- dse-0-node failed with ssl verification during Datastax-DSE upgrade from 2.1.2-5.1.2 to 2.2.0-5.1.2 on DC/OS 2.0.2. (COPS-6116)
+- CRDB was running and bouncer can not connect to it, because the bouncer's SELECT query fails. (COPS-5979)
```
Comment (cneth) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:65]: The PR in the COPS has a perfect description of what was changed to fix this ticket.
```suggestion
- Cache adminrouter permissions for 10s to reduce load on Bouncer in large clusters. (COPS-5979, COPS-5915, D2IQ-65296)
```

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 2]
```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
...
+- Nested Marathon groups were not inheriting enforceRole behavior from top-level groups. (COPS-6529)
+- enforceRole was automatically being set to false on all Marathon groups. (COPS-6533)
+- Selecting *Run* on a job or selecting *Delete* to remove a group had no effect and threw an exception. (COPS-6324) 
+- etcd provided an incorrect response in calicoctl. (COPS-6341)  
+- Users were unable to remove empty folders from Metronome. (COPS-6139) 
+- Hostname Resolution was failing for VIPs. (COPS-6411)  
+- Using file-based secrets caused mount failure and issues in the json editor. (COPS-6085, D2IQ-68114)  
+- dse-0-node failed with ssl verification during Datastax-DSE upgrade from 2.1.2-5.1.2 to 2.2.0-5.1.2 on DC/OS 2.0.2. (COPS-6116)
+- CRDB was running and bouncer can not connect to it, because the bouncer's SELECT query fails. (COPS-5979) 
+- An unknown response code was received when querying a DC/OS endpoint. (COPS-5915, D2IQ-65296)
```
Comment (cneth) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:66]: Delete this one and add the COPS/D2iQ numbers to the previous item as they are related:
```suggestion
```

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 3]
```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
...
+- DC/OS UI showed unexpected information in **Services->Tasks table->CPU** column. (COPS-6555)
+- Telgraf was consuming too much CPU. (COPS-5629)
+- UI failed to show manually assigned port on endpoints tab. (COPS-6491)
+- Exhibitor was writing JNA files to /tmp. (COPS-6111, D2IQ-68109) 
+- Renaming or deleting folders via the Jupyter UI resulted in a rename error and delete failed. (COPS-6166)
+- After an upgrade, the dcos-telegraf directories had incorrect permissions leading to a problem launching tasks. (COPS-6232, D2IQ-69295)
+- Exhibitor endpoint responses were inconsistent. (D2IQ-70393, COPS-6326)  
+- DC/OS OSS UI was not displaying a user name, but instead showed a *User added through OIDC ID Token login* message. (COPS-6295, D2IQ-70199) 
+- Added details of job to Metronome Overview page and Job Runs page. (COPS-4665)
+- Installer was setting Spartan IP's as the 'resolvers'. (COPS-4616)
```
Comment (cneth) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:76]:
```suggestion
- The DC/OS Installer now checks to make sure that the Spartan IP's are not set as the 'resolvers'. (COPS-4616)
```

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 4]
```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
...
+- A master node was not able to rejoin a cluster after failure/restart when another master was offline or being upgraded. (COPS-1754)
+- Turned on enable_docker_gc by default for all types of DC/OS installations. (COPS-5520)
+- Could not disable Calico in config.yaml. (COPS-6451)
```
Comment (cneth) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:79]:
```suggestion
- Added the ability to disable Calico in config.yaml. (COPS-6451)
```

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 5]
```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
...
-DC/OS now allows an operator to rotate the custom CA certificates by simply updating configuration settings during an upgrade. This feature ensures all the services that are using the custom CA based certificates are automatically updated after an upgrade.
+DC/OS now allows an operator to rotate the custom CA certificates by simply updating configuration settings during an upgrade. This feature ensures all the services that are using the custom CA based certificates are automatically updated after an upgrade. 
 
 # Breaking changes
 
 # Component Versions
 DC/OS 2.2.0 includes the following component version updates:
 
+- Marathon 1.11.16
+
```
Comment (vinodkone) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:41]: Let's update this:

-- Apache® Mesos® 1.11.0-dev
-- Marathon 1.11.23
-- Metronome 0.6.68
-- DC/OS UI  6.1.16
-- CockroachDB 19.1.11
-- Etcd 3.4.9

**Review by vinodkone — APPROVED** [review]
LGTM modulo Craig's excellent comments.

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 6]
```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
 DC/OS's Universal Container Runtime (UCR) now supports external volumes provided via the Container Storage Interface (CSI). Storage providers which integrate with the CSI specification provide plugins which DC/OS users may install into their cluster. These plugins allow volumes backed by that provider to be attached to task containers. This initial release of CSI support in DC/OS 2.2 has some caveats. For more information, see the [CSI](/mesosphere/dcos/2.2/storage/external-storage/csi/).
```
Comment (timcharper) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:28]:
```suggestion

## Marathon Support for Mesos Offer Constraints

Marathon can now send offer constraints to Mesos to reduce the number of offers it needs to decline due to placement constraints. This behavior is intended to improve the launch speed for services with placement constraints. It is currently experimental and disabled by default. To enable, add the line `MARATHON_MESOS_OFFER_CONSTRAINTS=` to the file `/var/lib/dcos/marathon/environment` on all masters, and restart Marathon.
```

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
 excerpt: Release notes for DC/OS 2.2, including Open Source attribution, and version policy.
 ---
-Mesosphere&reg; DC/OS&trade; 2.2 Beta 1 was released on 17, September 2020.
+Mesosphere&reg; DC/OS&trade; 2.2 was released on 29, October 2020.
 
-[button color="light" href="https://downloads.dcos.io/dcos/testing/2.2.0-beta1/commit/bc8a362bd72269db6fc0b0ac287dc8251d4b4b35/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+[button color="light" href="??"]Download DC/OS Open Source[/button]
 
-[button color="purple" href="https://downloads.mesosphere.io/dcos-enterprise/testing/2.2.0-beta1/commit/9d3b80b3206de4732ec81d8406f4ce1ac14a8406/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+[button color="purple" href="??"]Download DC/OS Enterprise* [/button]
```
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/mesosphere/dcos/2.2/release-notes/2.2/index.md]: done

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md (hunk 5 above)
Comment (cmastr) [reply pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:49]: done

**Pull request merged**